### ISSUE TITLE
Color visualization of the trend on the moving average.

### ISSUE THREAD

**Issue by NikolayNelubov** [highfestiva/finplot#26]
Hello. Is the color visualization of the trend on the moving average possible? Thanks for the help! 
![THV_Coral](https://user-images.githubusercontent.com/47115620/90754198-1f521b80-e2c9-11ea-8358-f792913040db.jpg)

**Comment by highfestiva** [comment]
Yes, but unfortunately you need three different lines: one green, one yellow and one red. This is part of the core of the library and is not going to change. Perhaps there is some way to facilitate this, but in such case it needs to support updating as well. I'll think about if or how such a thing could be accomplished.

**Comment by NikolayNelubov** [comment]
I understand, also thought about three lines. Is it possible to use separate objects instead of (lines), for example - triangles, circles? In theory, individual objects are easier to manipulate(combine into groups, change certain properties), but I suppose the computational load will increase. 
![Coral](https://user-images.githubusercontent.com/47115620/90798021-e6ce3400-e300-11ea-8971-263006e489ec.png)

**Comment by NikolayNelubov** [comment]
Perhaps you need to create a class that will greatly expand the capabilities of any types of moving averages.

For example **mov_averages(**date (input data), type(lines, circles, squares), acceleration factor(trend strength), uptrend color, downtrend color**)**. But the implementation is tricky. Need to think...
